src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -788,7 +789,7 @@ public void run() {
     final int xSpace = 5;
     final BufferedImage img = Util.createImage(categories.size() * (xSpace + icon_width),
         uiContext.getUnitImageFactory().getUnitImageHeight(), true);
-    final Graphics2D g = (Graphics2D) img.getGraphics();
+    final Graphics2D g = img.createGraphics();
```

Comment:
getGraphics ist just kept for backwards compatibility... createGraphics ist recommendet wince it creates a Graphics2D object